core/src/consensus/tendermint/message.rs

```diff
@@ -157,10 +141,9 @@ pub fn consensus_view(header: &Header) -> Result<View, ::rlp::DecoderError> {
     UntrustedRlp::new(view_rlp.as_slice()).as_val()
 }
 
-/// Proposal signature.
-pub fn proposal_signature(header: &Header) -> Result<Signature, ::rlp::DecoderError> {
-    UntrustedRlp::new(header.seal().get(1).expect("seal passed basic verification; seal has 3 fields; qed").as_slice())
-        .as_val()
+pub fn previous_block_view(header: &Header) -> Result<View, ::rlp::DecoderError> {
+    let view_rlp = header.seal().get(1).expect("seal passed basic verification; seal has 3 fields; qed");
```

Comment:
Isn't the second item of tendermint seal `cur_view`?

Reply:
Ignore the previous comment, I found that `seal_fields()` makes `cur_view` the first item.